CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# All the files under /.github/workflows belong to the sidestream team members to prevent malicious changes to the pipeline
+/.github/workflows/ @DeusAvalon @valiafetisov @BracketJohn
```

Comment:
@valiafetisov @BracketJohn I would have loved to use our `@powerhouse-inc/Sidestream` team but because the GitHub Team is a set to `secret` it cannot be used in a CO file.

See the other CO file in the `k8s-projects` repo:
https://github.com/sidestream-tech/k8s-projects/blob/main/CODEOWNERS

And i removed the SyncPolicy for the deployment to prevent auto deployments:
https://github.com/sidestream-tech/k8s-projects/commit/e3f60a612df830732fd480187480fe362aecbad1

Reply:
Thanks, I think it's enough for this PR